core/dao/sqldb/app.go

```diff
@@ -0,0 +1,225 @@
+package sqldb
+
+import (
+	"context"
+	"database/sql"
+	"errors"
+	"fmt"
+
+	"github.com/teamyapp/cloud/libs/errs"
+	"github.com/teamyapp/cloud/libs/telemetry"
+	"github.com/teamyapp/teamy-backend/core/dao"
+	"github.com/teamyapp/teamy-backend/core/entity"
+)
+
+var _ dao.App = (*App)(nil)
+
+type App struct {
+	dataCollector telemetry.DataCollector
+	db            *sql.DB
+}
+
+func (a App) FindAppByAppID(ct context.Context, appID uint64) (entity.App, *errs.Error) {
+	app := entity.App{}
+	err := a.db.QueryRow(`
+	SELECT
+	    id,
+	    apisecret,
```

Comment:
```suggestion
	    api_secret,
```
Can we rename this column to `api_secret` instead?

Reply:
I'll add that in the migration